src/include/ipc/dai.h

```diff
@@ -90,7 +90,9 @@ enum sof_ipc_dai_type {
 	SOF_DAI_AMD_SP,			/**< Amd SP */
 	SOF_DAI_AMD_DMIC,		/**< Amd DMIC */
 	SOF_DAI_AMD_HS,			/**< Amd HS */
-	SOF_DAI_MEDIATEK_AFE            /**< Mtk AFE */
+	SOF_DAI_MEDIATEK_AFE,           /**< Mtk AFE */
```

Comment:
Fixing this checkpatch warning is easy: don't add empty comments that merely repeat the code.

Reply:
![image](https://user-images.githubusercontent.com/101189005/201279264-487299cd-8642-4a4b-99b8-ab7bfe5f3611.png)